src/app/shared/auth/store/epics.ts

```diff
@@ -41,7 +41,7 @@ export const authEpics: Epics = {
         ]
       });
     }),
-    switchMap(() => appStorageService.token.get()),
+    switchMap(() => secureStorageService.token.get()),
```

Comment:
I think we could modify the `AppStorageService` to operate both secure- and async- storage items. Something like:
```ts
import { AsyncStorageItem, SecureStorageItem } from './models';

class AppStorageService {
  public token = new SecureStorageItem('token');
  public userID = new AsyncStorageItem('token');
}
```
So the app will still interact with the one storage service.

Thoughts?